_tutorials/2016-05-22-getting-started-on-xrv64.md

```diff
@@ -0,0 +1,101 @@
+---
+published: false
+date: "2016-05-22 13:49 -0700"
+title: Getting Started on XRv64
+permalink: "https://xrdocs.github.io/application-hosting/tutorials/xrv64-quickstart"
+author: Lisa Roach
+excerpt: "Getting started with Cisco's XRv64 Vagrant Instance."
+tags: 
+  - vagrant
+  - iosxr
+  - cisco
+  - XRv64
+---
+### Quick Start Using XRv64 Box image
+Quick guide to get up and running with XRv64 and vagrant/Virtualbox.
+ 
+#### Requirements:
+- [Vagrant](https://www.vagrantup.com/) and [Virtualbox](https://www.virtualbox.org/) installed on your laptop
+
+#### Single Node:
+
+	vagrant init xrv64
+  ![vagrant init xrv64](https://xrdocs.github.io/xrdocs-images/assets/tutorial-images/xrv64_vagrant_init.png)
```

Comment:
Ah bummer, I was trying to really avoid taking the screenshots because I can’t launch the multi-node setup on my laptop. I can do the single node launch screenshots, could you or someone else do the multi-node?

Sorry for the inconvenience, I actually just got approved for a Mac! So hopefully very soon this won’t be a problem anymore.


Lisa Roach
SOFTWARE SYSTEMS ENGINEER.US SALES


From: Akshat Sharma [mailto:notifications@github.com]
Sent: Tuesday, May 24, 2016 12:38 PM
To: xrdocs/application-hosting
Cc: Lisa Roach (lisroach); Author
Subject: Re: [xrdocs/application-hosting] Created 2016-05-22-getting-started-on-xrv64.md (#20)


In _tutorials/2016-05-22-getting-started-on-xrv64.md<https://github.com/xrdocs/application-hosting/pull/20#discussion_r64456074>:

> +tags:

> +  - vagrant

> +  - iosxr

> +  - cisco

> +  - XRv64

> +---

> +### Quick Start Using XRv64 Box image

> +Quick guide to get up and running with XRv64 and vagrant/Virtualbox.

> +

> +#### Requirements:

> +- [Vagrant](https://www.vagrantup.com/) and [Virtualbox](https://www.virtualbox.org/) installed on your laptop

> +

> +#### Single Node:

> +

> +     vagrant init xrv64

> +  ![vagrant init xrv64](https://xrdocs.github.io/xrdocs-images/assets/tutorial-images/xrv64_vagrant_init.png)

Unfortunately, these are Rich's screenshots which showcase XRv64 (internal names). Would it be possible for you to try it out on your windows setup and get screenshots from there? And use xrv or iosxrv as the name of your vagrant box.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/xrdocs/application-hosting/pull/20/files/d8c533664b4aeb754bff4eb7bf18407fb9e8eca6#r64456074>